evcache-client/src/main/java/com/netflix/evcache/pool/eureka/EurekaNodeListProvider.java

```diff
@@ -56,7 +56,14 @@ public EurekaNodeListProvider(ApplicationInfoManager applicationInfoManager, Eur
      */
     @Override
     public Map<ServerGroup, EVCacheServerGroupConfig> discoverInstances(String _appName) throws IOException {
-        if ((applicationInfoManager.getInfo().getStatus() == InstanceStatus.DOWN)) {
+        final Property<Boolean> ignoreAppEurekaStatus = props.get("evcache.ignoreAppEurekaStatus", Boolean.class).orElse(false);
+
+        if (ignoreAppEurekaStatus.get())
+            log.info("Not going to consider the eureka status of the application, to initialize evcache client.");
```

Comment:
i would recommend changing this to debug log, since this isn't a very common case of applications wanting to do useful work, while being down in discovery.

Reply:
This log gets printed only when someone has enabled the property. I wanted to ensure we print some INFO log to ensure we "capture" this if we are looking into some issue, so we know about the extra behavior.